### DISCUSSION TITLE
Any changes in the current nikobus configuration (.nkb)?

### DISCUSSION THREAD

**Discussion by jackolu** [fdebrus/Nikobus-HA#29] [General]
Will the existing nikobus configuration (.nkb) (as it functions now from 20 years on) be modified in any way?
After a Home Assistant outage, will everything still function identically as before the use of this integration?

I want to make sure that all light switches and programmed power-on times for the blinds remain the same, even if Home Assistant fails or I no longer want to use this add-on. Is it possible to revert everything back to the old state?

**Comment by fdebrus** [comment]
The integration will not in any shape or form touch your Nikobus configuration. It will remains as is.
You can safetly try the integration, Nikobus will continue to function as defined from 20y on. 

Now, you have to define your buttons and modules as they exist in Nikobus for the integration to "be aware" of the current config and perform as it shoud. This is a one time manual steps, as this data cannot be retrieved from Nikobus and has to be user provided. One time effort.

**Comment by jackolu** [comment]
Perfect! Thank you for your fast reply. Then I will feel confident enough to try out this integration in my parents' house ;)
